packages/server/src/fhir/operations/expunge.test.ts

```diff
@@ -193,6 +193,9 @@ async function existsInDatabase(tableName: string, id: string | undefined): Prom
 }
 
 async function existsInLookupTable(tableName: string, id: string | undefined): Promise<boolean> {
-  const rows = await new SelectQuery(tableName).column('id').where('resourceId', '=', id).execute(getDatabasePool());
+  const rows = await new SelectQuery(tableName)
+    .column('resourceId')
+    .where('resourceId', '=', id)
+    .execute(getDatabasePool());
```

Comment:
This is unsafe to deploy in the same PR as the DB migration: the migration will occur with old instances still running and potentially referencing the deleted column

Reply:
This is just a test.  The test was verifying that the rows were actually gone from the db.